TestCntlrApp/src/ueApp/ue_app.c

```diff
@@ -7775,73 +7776,68 @@ PUBLIC S16 ueUiProcErabsInfoMsg(Pst *pst, NbuErabsInfo *pNbuErabsInfo)
      ret = ueSendUeRadCapInd(ueCb);
    }
 
-   for (i = 0; i < pNbuErabsInfo->erabInfo->numOfErab; i++)
-   {
-      nasPdu = &pNbuErabsInfo->erabInfo->rabCbs[i].nasPdu;  
-      /* Decoding the PDU */
-      ret = ueAppEdmDecode(nasPdu, &ueEvnt);
-      if (ret != ROK)
-      {
-         UE_LOG_ERROR(ueAppCb, "NAS pdu decoding failed");
-         ret = RFAILED;
-         RETVALUE(ret);
-      }
-      if (ueEvnt == NULLP)
-      {
+   for (i = 0;
+     i < (pNbuErabsInfo->erabInfo && pNbuErabsInfo->erabInfo->numOfErab);
```

Comment:
please take the check on erabInfo outside the loop if there is a chance that it is not allocated.

Reply:
In order to validate memory is allocated, first condition, "pNbuErabsInfo->erabInfo" within for loop condition was added.
Yes it would more appropriate, if we add the null check condition before the loop; as it doesn't validate condition "pNbuErabsInfo->erabInfo->numOfErab" times

Reply:
In order to validate memory is allocated, first condition, "pNbuErabsInfo->erabInfo" within for loop condition was added.
Yes it would more appropriate, if we add the null check condition before the loop; as it doesn't validate condition "pNbuErabsInfo->erabInfo->numOfErab" times

Reply:
In order to validate memory is allocated, first condition, "pNbuErabsInfo->erabInfo" within for loop condition was added.
Yes it would more appropriate, if we add the null check condition before the loop; as it doesn't validate condition "pNbuErabsInfo->erabInfo->numOfErab" times

Reply:
In order to validate memory is allocated, first condition, "pNbuErabsInfo->erabInfo" within for loop condition was added.
Yes it would more appropriate, if we add the null check condition before the loop; as it doesn't validate condition "pNbuErabsInfo->erabInfo->numOfErab" times

Reply:
In order to validate memory is allocated, first condition, "pNbuErabsInfo->erabInfo" within for loop condition was added.
Yes it would more appropriate, if we add the null check condition before the loop; as it doesn't validate condition "pNbuErabsInfo->erabInfo->numOfErab" times